src/riak_test_escript.erl

```diff
@@ -297,26 +297,45 @@ run_test(Test, Outdir, TestMetaData, Report, HarnessArgs, NumTests) ->
         1 -> keep_them_up;
         _ -> rt:teardown()
     end,
-    CoverageFile = rt_cover:maybe_export_coverage(Test, CoverDir, erlang:phash2(TestMetaData)),
```

Comment:
Yeah, I'd thought that 80 was our agreed upon convention, but I'm more than willing to use 100 instead.